web/src/user_profile.js

```diff
@@ -321,6 +326,19 @@ export function get_custom_profile_field_data(user, field, field_types) {
     return profile_field;
 }
 
+function render_bot_profile_fields(bot) {
+    const user_type = {
+        name: bot.full_name,
+        should_add_guest_user_indicator: bot.is_guest,
+    };
+    $(".user_profile_name").html(render_user_full_name(user_type));
+
+    const $botOwnerField = $(".bot_owner_user_field");
```

Comment:
This should be renamed as `$bot_owner_field` for consistency.

Reply:
Fixed it.